data/src/main/java/com/microsoft/azure/kusto/data/BaseClient.java

```diff
@@ -15,12 +15,16 @@
 
 import java.io.IOException;
 import java.io.InputStream;
+import java.io.UncheckedIOException;
 import java.lang.invoke.MethodHandles;
+import java.time.Duration;
 import java.util.Optional;
+import java.util.concurrent.TimeUnit;
 
 public abstract class BaseClient implements Client, StreamingClient {
 
     private static final int MAX_REDIRECT_COUNT = 1;
+    private static final int CLIENT_SERVER_DELTA_IN_MILLISECS = (int) TimeUnit.SECONDS.toMillis(30);
```

Comment:
Maybe we should take this to change the name, EXTRA_TIMEOUT_FOR_CLIENT or something like that